package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-components",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-components/issues",
-  "version": "12.0.0",
+  "version": "12.1.0",
```

Comment:
I would opt for `12.0.1` since you're not adding any new functionality but only updating deps